Resources/config/menu.xml

```diff
@@ -13,6 +13,7 @@
         <service id="sonata.admin.menu.group_provider" class="Sonata\AdminBundle\Menu\Provider\GroupMenuProvider" public="false">
             <argument type="service" id="knp_menu.factory"/>
             <argument type="service" id="sonata.admin.pool"/>
+            <argument type="service" id="security.authorization_checker"/>
```

Comment:
This will fail in Symfony `2.3`. [`security.authorization_checker` was added in Symfony 2.6](http://symfony.com/blog/new-in-symfony-2-6-security-component-improvements).